src/video_recorder/recorder_support.js

```diff
@@ -150,7 +150,11 @@ Scoped.define("module:Recorder.Support", [
          * @private
          */
         __isCanvasBlank: function(canvas) {
```

Comment:
This function is only called by `_createSnapshot`. Would be best if we checked the width and height at the beginning of `_createSnapshot`, that way we can exit early avoiding running extra unnecessary code. 🙂

Reply:
Updated